src/app/app.component.ts

```diff
@@ -1,10 +1,11 @@
 import { Component } from '@angular/core';
+import { FormGroup, FormControl, FormBuilder, NgForm} from '@angular/forms';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'todoApp';
+  APP_TITLE : string = 'Add Tasks';
```

Comment:
Is there any specific reason to make APP_TITLE uppercase? Are we following any convention?

Reply:
I've tried to use uppercase for constants.